### ISSUE TITLE
Vampirism Open Terrain Generator (OTG) integration

### ISSUE THREAD

**Issue by darklysteamgear** [TeamLapen/VampirismIntegrations#30]
I know this mod adds in a new biome, why not have it's own biome file for integrating with other OTG mods such as far from home or biome bundle? 

Open Terrain Generator(OTG)
https://www.curseforge.com/minecraft/mc-mods/open-terrain-generator

Far From Home (FFH)
https://www.curseforge.com/minecraft/mc-mods/far-from-home

Biome Bundle(BB)
https://www.curseforge.com/minecraft/mc-mods/biome-bundle

it would be greatly appreciated

**Comment by maxanier** [comment]
I probably won't have time to investigate how OTG works (with modded biomes) and implement it. But if it just creating a certain file, maybe someone more experienced with that can do so or explain me what has to be done

**Comment by shadowbolt79** [comment]
Unless they make further changes with the 1.16.5 version, there's no way to include this automatically.

A biome file can certainly be created for it, and it would be able to pull from what your mod does, but it would have to manually be placed into the user's otg world files. For instance, ./mods/OpenTerrainGenerator/worlds/BiomeBundle/WorldBiomes/VampireForest.bc

Furthermore, that would not actually add the biome into the world. It would require additional edits to the world's WorldConfig.ini to add it into one or more of the biome groups.

In addition, in order to bring it up to the quality of several of the OTG presets without being meh, several biome objects would have to be created for custom tree generation, landmarks, formations, ect, and possibly several border biomes to get it to blend in with the other biomes nicely.

In Short, it could potentially be done by editing and adding files to specific presets, but not easily to OTG as a whole. It would best be done by someone experienced with making these presets, which I am not, yet.

**Comment by Cheaterpaul** [comment]
OTG is not available for recent versions of Minecraft

This issue can be reopened when the mod updates
